homeassistant/components/tradfri/base_class.py

```diff
@@ -102,12 +86,7 @@ class TradfriBaseDevice(TradfriBaseClass):
     def __init__(self, device, api, gateway_id):
         """Initialize a device."""
         super().__init__(device, api, gateway_id)
-        self._available = True
-
-    @property
-    def available(self):
-        """Return True if entity is available."""
-        return self._available
+        self._attr_available = True
```

Comment:
```suggestion
```
This is the default and should not be set here.

Reply:
Ups, corrected.